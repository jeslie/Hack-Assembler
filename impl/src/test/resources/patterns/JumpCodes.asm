// mnemonic // 111 0 101010 000 jmp
0;          //                  000
0;   JGT    //                  001
0;   JEQ    //                  010
0;   JGE    //                  011
0;   JLT    //                  100
0;   JNE    //                  101
0;   JLE    //                  110
0;   JMP    //                  111
