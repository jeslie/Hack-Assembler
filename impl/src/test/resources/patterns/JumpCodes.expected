1110101010000000
1110101010000001
1110101010000010
1110101010000011
1110101010000100
1110101010000101
1110101010000110
1110101010000111
